tests/integration/wsapi/test_consumer.py

```diff
@@ -567,6 +573,11 @@ def get_rulebook_process(instance_id):
     return models.RulebookProcess.objects.get(pk=instance_id)
 
 
+@database_sync_to_async
```

Comment:
@hsong-rh Isn't this https://github.com/ansible/eda-server/blob/500d4af032674f30b6a95af8e97ef6c8b920f69f/src/aap_eda/core/models/rulebook_process.py#L142

Reply:
Yes, we have to call it inside @database_sync_to_async